docusaurus/docs-internals/editor-controls/data-controls/number.md

```diff
@@ -0,0 +1,235 @@
+|    `config.step`    |                                                                        `number`                                                                         |   `1`   | The intervals value that will be incremented or decremented when using the controls’ spinners.                                                                                                                                                                    |
+|  `config.spinner`   |                                                                        `boolean`                                                                        | `true`  | Enables or disables the display of arrows that increment or decrement a number                                                                                                                                                                                    |
+| `config.updateRate` |                                                                        `number`                                                                         |  `16`   | The rate at which the component updates its value, specified in milliseconds. This can control how frequently changes to the input value are propagated                                                                                                           |
+|    `config.size`    |                                                          `short` / `medium` / `large` / `auto`                                                          | `short` | A string that specifies the size of the input field. Common values might include: .                                                                                                                                                                               |
+|  `helper.content`   |                                                                        `string`                                                                         |    -    | If provided, an icon is displayed next to the label. When hovering over this icon, a tooltip with additional information appears.                                                                                                                                 |
+|  `helper.position`  | `top-start` / `top` / `top-end` / `right-start` / `right` / `right-end` / `bottom-end` / `bottom` / `bottom-start` / `left-end` / `left` / `left-start` |  `top`  | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                |
+
+### Return value
+
+The return value is "number".
```

Comment:
The return value is "number". -> The return value is a `number` representing the text field value.